PZPullToRefresh/PZPullToRefreshView.swift

```diff
@@ -132,11 +133,17 @@ public final class PZPullToRefreshView: UIView {
     public func refreshLastUpdatedDate() {
         if isShowUpdatedTime {
             if let date = delegate?.pullToRefreshLastUpdated(self) {
-                let formatter = NSDateFormatter()
-                formatter.AMSymbol = "AM"
-                formatter.PMSymbol = "PM"
-                formatter.dateFormat = "yyyy/MM/dd/ hh:mm:a"
-                lastUpdatedLabel?.text = "Last Updated: \(formatter.stringFromDate(date))"
+                var lastUpdateText:String
+                if (self.lastUpdatedLabelCustomFormatter != nil){
+                    lastUpdateText = self.lastUpdatedLabelCustomFormatter!(date: date)
```

Comment:
@dannyshmueli 
I think it's better to use `if let` like this.
```swift
if let formatter = self.lastUpdatedLabelCustomFormatter {
    lastUpdateText = formatter
} else {
...
```
What do you think it?